rpctest/README.md

```diff
@@ -16,28 +19,15 @@ This package was designed specifically to act as an RPC testing harness for
 any project wishing to programmatically drive a `btcd` instance of its
 systems/integration tests. 
 
-## Documentation
-
-[![ISC License]
-(http://img.shields.io/badge/license-ISC-blue.svg)](http://copyfree.org)
-[![GoDoc](https://img.shields.io/badge/godoc-reference-blue.svg)]
-(http://godoc.org/github.com/btcsuite/btcd/rpctest)
-
-Full `go doc` style documentation for the project can be viewed online without
-installing this package by using the GoDoc site
-[here](http://godoc.org/github.com/btcsuite/btcd/rpctest).
-
-You can also view the documentation locally once the package is installed with
-the `godoc` tool by running `godoc -http=":6060"` and pointing your browser to
-http://localhost:6060/pkg/github.com/btcsuite/btcd/rpctest
-
-
 ## Installation and Updating
 
 ```bash
 $ go get -u github.com/btcsuite/btcd/rpctest
 ```
 
 ## License
-Package rpctest is licensed under the [copyfree](http://copyfree.org) ISC
+
+
+Package blockchain is licensed under the [copyfree](http://copyfree.org) ISC

```

Comment:
Left over from my copy-pasta :)

Fixed.

On Fri, Aug 19, 2016 at 4:49 PM Dave Collins <notifications@github.com>
wrote:

> In rpctest/README.md
> <https://github.com/btcsuite/btcd/pull/371#discussion_r75556424>:
>
> >  ## Installation and Updating
> >
> >  ```bash
> >  $ go get -u github.com/btcsuite/btcd/rpctest
> >  ```
> >
> >  ## License
> > -Package rpctest is licensed under the [copyfree](http://copyfree.org) ISC
> > +
> > +
> > +Package blockchain is licensed under the [copyfree](http://copyfree.org) ISC
>
> blockchain -> rpctest
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/btcsuite/btcd/pull/371/files/5cf26fed3079e4c59c53b771eac19b400163c162..b869f4113accc48c46652219478fe57c7d297f8b#r75556424>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA87Lke0NueHGU-xFQBEuV_GdzP0d056ks5qhiTrgaJpZM4D7zxE>
> .
>